README.md

```diff
@@ -36,6 +36,9 @@ $ buildaur -S pokete-git
 
 For windows first install pynput and then do a Windows equivalent to the above.
 
+## Usage
+For non gameplay related usage see `--help`.
+
```

Comment:
Maybe add options here as well or put in that the game can be run by not supplying any options.